### PR TITLE
Disallows bitlist duplicates in seen_bits cache

### PR DESCRIPTION
**Pull request by farazdagi**
**What type of PR is this?**

Bug fix

**What does this PR do? Why is it needed?**
- If you push the very same attestation multiple times, it gets added to "hasSeen" cache multiple times, which is unnecessary.
- If item is already in cache, cache doesn't grow - but expiration time gets updated (so cache will remember the item for a longer period of time)

**Which issues(s) does this PR fix?**

N/A

**Other notes for review**
- cc @terencechain

### CODE REVIEW

File: beacon-chain/operations/attestations/kv/seen_bits.go [hunk 1]
```diff
@@ -19,7 +19,16 @@ func (p *AttCaches) insertSeenBit(att *ethpb.Attestation) error {
 		if !ok {
 			return errors.New("could not convert to bitlist type")
 		}
-		seenBits = append(seenBits, att.AggregationBits)
+		alreadyExists := false
+		for _, bit := range seenBits {
+			if bit.Len() == att.AggregationBits.Len() && bit.Contains(att.AggregationBits) {
+				alreadyExists = true
+				break
```
Comment (nisdas) [inline beacon-chain/operations/attestations/kv/seen_bits.go:26]: if it already exists we should exit early then

Comment (farazdagi) [reply beacon-chain/operations/attestations/kv/seen_bits.go:26]: I deliberately do not exit -- and update the expiry time, so the cache remembers given attestation longer (supposing that if we had one duplicate, there may be another coming). Thoughts?

Comment (nisdas) [reply beacon-chain/operations/attestations/kv/seen_bits.go:26]: Dont feel too strongly on it if the purpose is to keep it alive in the cache longer. I am not too familiar with the code here so maybe @terencechain can give a better opinion on this.

Comment (farazdagi) [reply beacon-chain/operations/attestations/kv/seen_bits.go:26]: Agree! @terencechain , please, let me know if that updating of expiry time is even necessary? I did it this way assuming when we had one duplicate we might get more and expiry is to be tracked from the time of the last duplicate received.

If that update to expiry time is unnecessary, then Nishant's suggestion to exit early is spot on.

**Review by prestonvanloon — APPROVED** [review]
LGTM. I removed "OK to merge" label until @terencechain can respond to feedback questions

File: beacon-chain/operations/attestations/kv/seen_bits.go (hunk 1 above)
Comment (terencechain) [reply beacon-chain/operations/attestations/kv/seen_bits.go:26]: Both are likely fine. Updating the expiry time is more lenient in terms of validator profitability in some extreme cases and the performance impact is negligible. Hence I'm leaning towards `update` versus `exit`.

**Review by terencechain — APPROVED** [review]
lgtm

**Pull request merged**